internal/postprocessor/config.yaml

```diff
@@ -1292,6 +1269,11 @@ skip-module-scan-paths:
   # ignore the root module
   - .
   # Librarian released modules
+  - eventarc
```

Comment:
Do we want to keep this list alpha sorted?  It may make it easier to review the list of migrated APIs.